roadmap/implementers-guide/src/types/candidate.md

```diff
@@ -219,8 +219,8 @@ This struct encapsulates the outputs of candidate validation.
 struct ValidationOutputs {
 	/// The head-data produced by validation.
 	head_data: HeadData,
-	/// The validation data, persisted and transient.
-	validation_data: ValidationData,
+	/// The validation data, persisted.
+	validation_data: PersistedValidationData,
```

Comment:
Can you elaborate on this change?

Reply:
The reason is that the `ValidateCandidateExhaustive` API supports not providing the `TransientValidationData` if not desired. Then there would be no way to return it with the validation outputs.

However, this change also doesn't impact our other primary user of the `CandidateValidation` subsystem: candidate backing. Since Candidate Validation checks the transient validation data when validating from the runtime state, there isn't anything else that candidate backing could meaningfully do with that data.